fusor-ember-cli/app/templates/components/base-f.hbs

```diff
@@ -18,17 +18,17 @@
 
       <div class="{{inputClassSize}}">
         {{yield}}
-      </div>
```

Comment:
this change is just moving this `</div>` from here to line 31 
This makes the 'help-inline' block look prettier.